app/src/main/java/com/framgia/oleo/screen/follow/FollowPagerAdapter.kt

```diff
@@ -0,0 +1,33 @@
+package com.framgia.oleo.screen.follow
+
+import android.content.Context
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.FragmentManager
+import androidx.fragment.app.FragmentPagerAdapter
+import com.framgia.oleo.R
+import com.framgia.oleo.screen.followed.FollowedFragment
+import com.framgia.oleo.screen.followed.FollowingFragment
+import com.framgia.oleo.screen.followed.WaitingFragment
+import com.framgia.oleo.utils.Index
+
+class FollowPagerAdapter(fm: FragmentManager, context: Context) : FragmentPagerAdapter(fm) {
+    private val tabTitles = context.resources.getStringArray(R.array.tab_titles)
+
+    override fun getItem(position: Int): Fragment {
+        when (position) {
+            Index.POSITION_ZERO -> return FollowedFragment.newInstance()
```

Comment:
tạo biến hằng riêng đặt tên cho dễ nhận biết e 
```suggestion
            TAB_FOLLOW -> return FollowedFragment.newInstance()
```